cli/internal/context/context.go

```diff
@@ -129,13 +144,13 @@ func isWorkspaceReference(packageVersion string, dependencyVersion string, cwd s
 
 // SinglePackageGraph constructs a Context instance from a single package.
 func SinglePackageGraph(repoRoot turbopath.AbsoluteSystemPath, rootPackageJSON *fs.PackageJSON) (*Context, error) {
-	packageInfos := make(map[interface{}]*fs.PackageJSON)
-	packageInfos[util.RootPkgName] = rootPackageJSON
+	workspaceInfos := make(graph.WorkspaceInfos)
+	workspaceInfos[util.RootPkgName] = rootPackageJSON
```

Comment:
https://github.com/uber-go/guide/blob/master/style.md#initializing-maps
```suggestion
	workspaceInfos := map[string]*fs.PackageJSON{util.RootPkgName: rootPackageJSON}
```